src/networkremote/clientmanager.h

```diff
@@ -0,0 +1,29 @@
+#ifndef CLIENTMANAGER_H
+#define CLIENTMANAGER_H
+
+#include <QObject>
+#include <QTcpSocket>
+#include <QList>
+
+class Application;
+class NetworkRemoteClient;
+
+class NetworkRemoteClientManager : public QObject
+{
+  Q_OBJECT
+public:
```

Comment:
Indent 1 space, same problem for all the code.

Reply:
Done